EU4toV2/Source/Mappers/CultureGroups/CultureGroups.cpp

```diff
@@ -44,55 +44,57 @@ mappers::CultureGroups::CultureGroups(std::istream& theStream)
 
 void mappers::CultureGroups::registerKeys()
 {
-	registerRegex("\\w+", [this](const std::string& cultureGroupName, std::istream& theStream)
+	registerRegex("\\w+", [this](const std::string& cultureGroupName, std::istream& theStream) {
+		std::vector<Culture> cultures;
+		CultureGroup newGroup(cultureGroupName, theStream);
+		if (cultureGroupsMap.count(cultureGroupName))
 		{
-			std::vector<Culture> cultures;
-			CultureGroup newGroup(cultureGroupName, theStream);
-			if (cultureGroupsMap.count(cultureGroupName))
+			// We would normally override base definitions with incoming CK2 ones, but CK2 definitions
+			// are crap and don't actually list all required cultures, so we have to merge.
+			for (const auto& cultureItr: newGroup.getCultures())
 			{
-				// We would normally override base definitions with incoming CK2 ones, but CK2 definitions
-				// are crap and don't actually list all required cultures, so we have to merge.
-				for (const auto& cultureItr: newGroup.getCultures())
-				{
-					cultureGroupsMap[cultureGroupName].mergeCulture(cultureItr.first, cultureItr.second);
-				}
+				cultureGroupsMap[cultureGroupName].mergeCulture(cultureItr.first, cultureItr.second);
 			}
-			else cultureGroupsMap.insert(std::make_pair(cultureGroupName, newGroup));
-		});
+		}
+		else
+			cultureGroupsMap.insert(std::make_pair(cultureGroupName, newGroup));
```

Comment:
`emplace` would work here, AFAIK it's more efficient.

Reply:
I didn't even edit this file, it's just reclang speaking. I'll get to it in the next pass.